### DISCUSSION TITLE
Explicit albums

### DISCUSSION THREAD

**Discussion by MadDucks** [RandomNinjaAtk/docker-lidarr-extended#45] [Ideas]
Can you create a fourth choice that would search for clean and if it can't find it looks for explicit?

**Comment by RandomNinjaAtk** [comment]
No, explicit albums typically have a clean version. So it's not necessary.

**Reply by dougalbob** [reply]
Ive got to be honest I sort of feel like the OP on this one,  I'm no prude but given the choice I'd rather have a clean version by default and the occasional explicit version where clean does not exist. If the search then picked up the explicit where no clean existed I could make a judgment on a per cases basis as to whether it should remain in the library.   If there is some technical reason why adding this causes too much complexity I completely understand and remain massively thankful for what we already have. Take care

**Reply by RandomNinjaAtk** [reply]
Again, I would think if an explicit album exists, a clean version should also exist... Hence using the clean lyric option, should only grab all clean versions of explicit albums and regular clean releases. Thus, what your asking for is not needed.... because it can already do that...

**Reply by MadDucks** [reply]
As I've been looking at my collection, I'm finding instances where that's not the case,  a prime example is Pale Waves: https://www.deezer.com/us/artist/11940941  All their albums show as explicit, there are no clean albums.   I've got a few other examples of this (T.I, G-Eazy).   

Now, what I can do, now that lidarr-extended has gone thru my library, is turn on "both" and see what it grabs.  It just seems sub optimal, given that I'll have to do full scans again, and then watch when it goes thru the explicit albums to turn clean back on.

**Reply by RandomNinjaAtk** [reply]
Most people should only use it as "both", the only reason lyric type was added, is because someone asked for clean only setting and it was to accommodate the request.....
